README.adoc

```diff
@@ -299,11 +298,12 @@ servers:
 
 // Later this should be changed to:
 // For more information about which elements you can filter, see the MicroProfile 1.3 API.
+// https://openliberty.io/docs/ref/microprofile/1.3/
 For more information about which elements you can filter, see the `OASFilter`
 https://github.com/eclipse/microprofile-open-api/blob/master/api/src/main/java/org/eclipse/microprofile/openapi/OASFilter.java[implementation].
 
 To learn more about MicroProfile Config, visit the MicroProfile GitHub https://github.com/eclipse/microprofile-config[repository]
-and try one of the MicroProfile Config guides.
+and try one of the https://openliberty.io/guides/?search=Config[MicroProfile Config guides].
```

Comment:
Super small thing that you can take or leave...

If we made just the word "repository" a hyperlink, should we make just the word "guides" a hyperlink, too? Or hyperlink the entirety of "MicroProfile GitHub repository"?

Reply:
oh yes, you're right, I'll change that